src/Symfony/Component/Intl/Tests/Transliterator/EmojiTransliteratorTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Intl\Tests\Transliterator;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Intl\Transliterator\EmojiTransliterator;
+
+class EmojiTransliteratorTest extends TestCase
+{
+    public function testTransliterate()
+    {
+        $tr = EmojiTransliterator::getInstance('fr');
+
+        $this->assertSame(
+            'a grinningcat, blackcat, and a lion go to nationalpark... smilingfacewithheart-eyes partypopper yellowheart',
```

Comment:
I'm not sure to understand why we have english transliterations with 'fr' instance :thinking: